media/admin/com_virtuemart/models/zasilkovna_src/VirtueMartModelZasilkovna/Order/Order.php

```diff
@@ -100,6 +100,24 @@ class Order
     /* @var int $tax_id */
     private $tax_id;
 
+    /* @var \DateTime $created_on */
```

Comment:
Je tam DateTime i v případě starších verzí VM, Joomly, PHP?

Reply:
DateTime byl pridan do PHP verze 5.2. Nejstarsi Joomla jede myslim na PHP 5.3.10 a VirtueMart funguje od Joomla 3 a vys, tudiz ja bych rekl, ze by tam mel byt. Take, pokud se nepletu mi bylo receno, ze mam resit pouze Joomla4 s VirtueMart4, co se tyce kompatibility. Tak snad jsem to pochopil spravne.